liquid-rust-core/src/ir.rs

```diff
@@ -88,6 +88,8 @@ pub enum BinOp {
     Sub,
     Mul,
     Div,
+    Mod,
```

Comment:
This should be called `Rem` for consistency with `mir`. I think this also helps to differentiate it with `Mod` in types which has different semantics.

```suggestion
    Rem,
```

Reply:
Done.